packages/css-library/src/stylesheets/base/_va.scss

```diff
@@ -347,5 +347,8 @@ article > h1 {
 
   p {
     line-height: $lead-line-height;
+    font-size: $lead-font-size;
+    font-weight: $font-weight-normal;
+    line-height: $lead-line-height;
```

Comment:
This was added because `.va-introtext p` has custom font sizing and we want to maintain that. Otherwise it was being overriden by the USWDS normalization font size.

Reply:
Can the duplicate line-height be removed here?

Reply:
oh derp. Yes, good catch!